applications/GeoMechanicsApplication/tests/test_helper.py

```diff
@@ -502,6 +502,8 @@ def _process_nodal_result(self, words):
             value["value"] = float(words[1])
         elif self.result_type == "Vector":
             value["value"] = [float(x) for x in words[1:]]
+        elif self.result_type == "Matrix":
+            value["value"] = [float(x) for x in words[1:]]
```

Comment:
I would suggest to combine these two tests to show that they are treated in the same way:
```suggestion
        elif self.result_type == "Vector" or self.result_type == "Matrix":
            value["value"] = [float(x) for x in words[1:]]
```

Reply:
Good point, done